src/components/elements/action-link.tsx

```diff
@@ -1,4 +1,5 @@
-import Link, {LinkProps} from "@components/elements/link"
+import Link from "next/link"
```

Comment:
the `Link` from our elements has some logic to handle various link urls like absolute or drupal links. We will either need to add that logic in here or use the component from the elements.